docs/src/whatsnew/3.2.rst

```diff
@@ -25,8 +25,8 @@ This document explains the changes made to Iris for this release
    any issues or feature requests for improving Iris. Enjoy!
 
 
-v3.2.1 |build_date| [unreleased]
-********************************
+v3.2.1 (11 Mar 2022)
+====================
```

Comment:
We need to remember to make this fix in `main` too - for `dev.rst` and the template.

Reply:
The template does have the correct heading underlining,
![image](https://user-images.githubusercontent.com/8101163/157893501-3d3f2756-aa24-40ec-8678-8f3b2482f59c.png)
I must have just made an error when editing the template

Reply:
Oh yeah, I was looking at the wrong heading.